plugin-traffic/gradle.properties

```diff
@@ -2,4 +2,5 @@ VERSION_NAME=0.11.0-SNAPSHOT
 POM_ARTIFACT_ID=mapbox-android-plugin-traffic-v9
 POM_NAME=Mapbox Android Traffic Plugin
 POM_DESCRIPTION=Mapbox Android Traffic Plugin
-POM_PACKAGING=aar
\ No newline at end of file
+POM_GROUP_ID=com.mapbox.mapboxsdk
```

Comment:
it seems the group id is the same for all the modules, we could have it in the artifact-settings.gradle